src/test/resources/ConnectionParameters.properties

```diff
@@ -4,8 +4,11 @@ Project-ID=cdf-athena
 windowSize=1920x1040
 CDFURL=http://localhost:11011/pipelines/ns/default/studio
 
-BigQuery=https://pantheon.corp.google.com/bigquery?project=sap-adapter  
 
 #GCSbucket
 @TC-Demo-1_GCS=gs://cdf-athena-test/test.csv
-@TC-Demo-2_GCS=gs://cdf-athena-test/test-DataTypes.csv
\ No newline at end of file
+@TC-Demo-2_GCS=gs://cdf-athena-test/test-DataTypes.
```

Comment:
Is "no extension" expected?

Reply:
Added extension. We are not using this file as we have only TC1. Just kept that for reference check if it is working with multiple files.